apps/machine/zynqmp/zynqmp_linux_r5_proc.c

```diff
@@ -187,8 +204,12 @@ static int zynqmp_linux_r5_proc_notify(struct remoteproc *rproc, uint32_t id)
     prproc = rproc->priv;
 
     /* TODO: use IPI driver instead and pass ID */
```

Comment:
with you Pr is this comment is still valid ?

Reply:
nope good catch! updated in new commits